taskcluster/kinds/beetmover/kind.yml

```diff
@@ -0,0 +1,135 @@
+    - distillation-parallel-src-extract-best
+    - distillation-parallel-src-translate
+    - distillation-parallel-src-translations-score
+    - distillation-student-model-finetune
+    - distillation-student-model-quantize
+    - distillation-student-model-train
+    - evaluate
+    - evaluate-quantized
+    - evaluate-teacher-ensemble
+    - export
+    - train-teacher-model
+
+tasks:
+    beetmover:
+        label: beetmover-{name}
+        description: upload artifacts
+        from-deps:
+            group-by: single
+            # This will pull in stage, src_locale, trg_locale, etc. attributes
+            copy-attributes: true
+            set-name: retain-kind
+        cache:
+            type: upload
+            name: beetmover3
+            digest-data: []
+    
+        task-context:
+            from-parameters:
+                src_locale: training_config.experiment.src
+                trg_locale: training_config.experiment.trg
+                experiment_name: training_config.experiment.name
+                training_config_bucket: training_config.taskcluster.upload-bucket
+            substitution-fields:
+                - attributes
+                - worker.artifact-map
+                - worker.bucket
+                - label
+                - cache.name
+        worker-type: beetmover-translations
+        run-on-tasks-for: []
+        worker:
+            app-name: translations
+            # the `dep` bucket pushes to the non-production GCS bucket
+            # (moz-fx-translations-data--5f91-stage-translations-data)
+            # and should be used to push artifacts that don't need to live
+            # indefinitely. PRs and manual training runs from PRs fall into
+            # this category, and thus we hardcode it here to avoid, eg:
+            # manual training runs from PRs accidentally uploading to the
+            # release bucket. Sometimes other manual training runs can also
+            # be in this category (eg: for testing work on `dev-` branches).
+            # the `release` bucket pushes to the production GCS bucket
+            # (moz-fx-translations-data--303e-prod-translations-data) and
+            # should be used when preserving artifacts indefinitely is
+            # important. This is the default in the training config but can
+            # be adjusted when needed.
+            bucket:
+                by-tasks-for:
+                    github-pull-request: dep
+                    pr-action: dep
+                    default: "{training_config_bucket}"
+            # upstream artifacts defines which artifacts from upstream tasks we want to upload
+            upstream-artifacts:
+                by-upstream-kind:
+                    # Tasks from kinds matching these patterns will have all artifacts
+                    # uploaded; other steps will only have logs uploaded.
+                    (distillation-student.*|evaluate.*|export|train-teacher.*):
```

Comment:
+ train backwards

I would also include artifacts for corpus continuation here right away, but we should double-check with @gregtatum if he wants to do it separately.

The steps would be:
- corpus-align*
- distillation-corpus-final-filtering
- Maybe some other corpus producing steps if we want to rerun alignments (dependencies of the corpus-align* steps)

Reply:
@gregtatum - just reflagging you here: do we want non-log artifacts uploaded for any of the steps listed above? (Or any other ones, for that matter.)